src/fundus/publishers/us/occupy_democrats.py

```diff
@@ -41,7 +41,7 @@ def title(self) -> Optional[str]:
 
         @attribute
         def topics(self) -> List[str]:
-            return generic_topic_parsing(self.precomputed.ld.get_value_by_key_path(["Article", "keywords"]))
+            return generic_topic_parsing(self.precomputed.ld.xpath_search("Article/keywords", scalar=True))
```

Comment:
Also here, is the `scalar=True` intentional?

Reply:
Yes. Here we're searching for just one field `Article/keywords`. The field can still be a list. Scalar doesn't make any assumption about the type of the returned result, only about the number of results. With `scalar` we basically say that we expect only one field to match the expression.